app/assets/stylesheets/components/_contents-list.scss

```diff
@@ -0,0 +1,67 @@
+.app-c-contents-list {
+  // Always render the contents list above a
+  // back to contents link
+  position: relative;
+  margin-bottom: $gutter-two-thirds;
+  z-index: 1;
+  background: $white;
+  box-shadow: 0 20px 15px -10px $white;
+}
+
+// Remove underlines from lists of links
+// to improve legibility
+.app-c-contents-list__link {
+  text-decoration: none;
+
+  &:hover,
+  &:focus,
+  &:active {
+    text-decoration: underline;
+  }
+
+  .app-c-contents-list__list-item--parent > & {
+    font-weight: bold;
+  }
+}
+
+.app-c-contents-list__list-item {
+  padding-top: $gutter / 4;
+  line-height: 1.3;
+  list-style-type: none;
+}
+
+.app-c-contents-list__list-item--dashed {
+  $contents-spacing: $gutter-half + 5;
+  margin-left: $contents-spacing;
+  padding-right: $contents-spacing;
+
+  &:before {
+    content: "— ";
```

Comment:
Tried doing this quickly. It's a little fiddly. We want the em dash to be in the top left and not have text wrapping beneath it, which means wrapping it in a span. Then we need some logic to include/exclude that span based on whether dashes are being used. It felt wrong moving inheritance out of the stylesheet.